### ISSUE TITLE
Buckets are not actually optional

### ISSUE THREAD

**Issue by nilox94** [erm/mangum-cli#13]
When I execute
```
mangum init <name>
mangum package
```
with no `bucket-name` I get an error at the command `magnum package`.
It happens because the subprocess `aws cloudformation package ...` is executed at `mangum_cli/config.py:109` with a `None` object as the last parameter (corresponding to the unset bucket name).

**Comment by nilox94** [comment]
Indeed, the parameter `--s3-bucket` is not optional for the command `aws`, so it should be required.

**Comment by sig9org** [comment]
Hi, @nilox94.

It is optional at initialization and can be changed later. However, I think it is essential for packaging.

Is it better to be required at initialization?

**Comment by nilox94** [comment]
@sig9org, if the param `<bucket-name>` is intended to be optional, then the subcommand `package` should explicitly fail and alert about the unset parameter.
Now, let me answer your question (and I'm sorry about it ;) with another question.
Is there a scenario (or subcommand) in which `magnum` performs some "useful" task that doesn't need to configure a `<bucket-name>`?

**Comment by jordaneremieff** [comment]
@nilox94 your suggestion to make the subcommand `package` explicitly fail is probably what we want here with the current behaviour. The bucket name being optional during `init` is for the case in which a user wants to generate a configuration and manually edit the file to include a bucket name later.
